WordPress/jetpack_metadata/release_notes.txt

```diff
@@ -1,4 +1,7 @@
-- Font size and line height are now supported for block themes.
-- Add new stat cards in the Settings menu on the Stats screen.
-- When changing the media in an Image block that’s also a post’s Featured Image, get a prompt to replace the Featured Image.
-- Share text, images, and video in the app using your device’s Share action.
+* [*] Comment editor: Notification comments support in the new editor [https://github.com/wordpress-mobile/WordPress-Android/pull/15884]
+* [*] Site creation: Fixed bug where sites created within the app were not given the correct time zone, leading to post scheduling issues. [https://github.com/wordpress-mobile/WordPress-Android/pull/15904]
```

Comment:
Doubt: Is site creation applicable to Jetpack? 🤔

Reply:
Same question as https://github.com/wordpress-mobile/WordPress-Android/pull/15986#discussion_r811115368 then 😅

Reply:
Personally, I think we can remove  `Site creation: Fixed bug where sites created...` entry for JP app for the same reasons as https://github.com/wordpress-mobile/WordPress-Android/pull/15986#discussion_r811115368. 😅

Reply:
Addressed in [681333f](https://github.com/wordpress-mobile/WordPress-Android/pull/15986/commits/681333ff5bee95e216234470184fd4363154d507)